app/controllers/planning_applications/assessment/requirements_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class RequirementsController < BaseController
+      before_action :set_requirements, only: %i[index create]
+
+      def index
+        @categories = LocalAuthority::Requirement.categories
+        @existing_requirements = @planning_application.requirements.pluck(:description)
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def create
+        @categories = LocalAuthority::Requirement.categories
+        @requirements.where(id: params[:requirement_ids]).find_each do |requirement|
+          application_requirement = @planning_application.requirements.new(
+            description: requirement.description,
+            guidelines: requirement.guidelines,
+            url: requirement.url,
+            category: requirement.category
+          )
+          application_requirement.save
```

Comment:
It might be better to use the `!` version here, because that'll throw an exception if it fails (the version without `!` just returns true or false, but we're not checking the return value).

Reply:
```suggestion
          application_requirement.save!
```